### ISSUE TITLE
Search doesn’t work at the moment

### ISSUE THREAD

**Issue by jancborchardt** [owncloud/maps#21]
When searching for »Berlin«, I get:
```
GET apps/maps/search?search=Berlin&bbox=[…] 500 (Internal Server Error)
    mapSearch.getSearchResults script.js line 337
    (anonymous function) script.js line 272
```

@brantje

**Comment by DJaeger** [comment]
@brantje: Have you fixed this?
If not, would you share your findings, that someone else can fix?

**Comment by v1r0x** [comment]
Is this still present? I don't get any errors.

**Comment by jancborchardt** [comment]
Search still doesn’t really work. No suggestions are shown, the log says:

```
Object { contacts: Array[17], nodes: Array[0], addresses: Array[10] } script.js:449:4
```

And in the owncloud.log:
```
Attempt to assign property of non-object at \/home\/jan\/owncloud\/apps\/maps\/controller\/pagecontroller.php#194"
"url":"\/owncloud\/index.php\/apps\/maps\/search?search=Berlin&bbox=13.449797630310059%2C52.50692273870633%2C13.480696678161621%2C52.52269704703951
```

**Comment by v1r0x** [comment]
Suggestions never worked for me. Normal search with searching on pressing return/enter worked.
I'm currently trying to integrate the search bar into the map (using a new version of (I think) the one which should show suggestions right now) and add the functionality of #8 to it. Maybe this could solve all search related issues

**Comment by jancborchardt** [comment]
Cool! Yeah, that would probably work.

**Comment by v1r0x** [comment]
the current status is in the testing branch of my fork. If you have the time, you could have a look at it.

**Comment by jancborchardt** [comment]
@v1r0x if you just submit a pull request for that, it’s easier to test, review and then merge. :)

**Comment by v1r0x** [comment]
@jancborchardt will do that after my holiday. The branch was for @Henni , because I have a weird problem (function only called if there is an alert()). I just told you, if you want to have a look at the current progress ;)

**Comment by DJaeger** [comment]
Any news or still at holdidays? ;-)

**Comment by v1r0x** [comment]
No, back from my holiday...sadly :D
I think I should talk to @Henni or anyone else to figure out the problem. I set up a local test environment on my laptop, which should be easier to debug.
Currently I have a side project, associated with OSM/OC Maps, so I don't work 100% on maps itself, but I will definitely finish this soon ;)

**Comment by v1r0x** [comment]
Still haven't found the problem...If anyone is interested or has an idea: https://github.com/v1r0x/maps/tree/testing (https://github.com/v1r0x/maps/blob/testing/js/3rdparty/leaflet/lib/Control.Geocoder.js#L307)

**Comment by jancborchardt** [comment]
This seems to be resolved, and only #38 (zooming in on the best result) remains.
